src/main/java/org/sunbird/workflow/service/impl/WorkflowServiceImpl.java

```diff
@@ -1245,4 +1249,260 @@ public SBApiResponse workflowBulkUpdateTransitionV1(String userAuthToken, Multip
+		cell0.setCellValue("Full Name");
+
+		Cell cell1 = row.createCell(1);
+		cell1.setCellValue("Email");
+
+		Cell cell2 = row.createCell(2);
+		cell2.setCellValue("Mobile Number");
+
+		Cell cell3 = row.createCell(3);
+		cell3.setCellValue("Group");
+
+		Cell cell4 = row.createCell(4);
+		cell4.setCellValue("Designation");
+
+		Cell cell5 = row.createCell(5);
+		cell5.setCellValue("Gender");
+
+		Cell cell6 = row.createCell(6);
+		cell6.setCellValue("Category");
+
+		Cell cell7 = row.createCell(7);
+		cell7.setCellValue("Date of Birth (dd-mm-yyy)");
+
+		Cell cell8 = row.createCell(8);
+		cell8.setCellValue("Mother Tongue");
+
+		Cell cell9 = row.createCell(9);
+		cell9.setCellValue("Employee ID");
+
+		Cell cell10 = row.createCell(10);
+		cell10.setCellValue("Office Pin Code");
+
+		Cell cell11 = row.createCell(11);
+		cell11.setCellValue("External System ID");
+
+		Cell cell12 = row.createCell(12);
+		cell12.setCellValue("External System Name");
+
+		Cell cell13 = row.createCell(13);
+		cell13.setCellValue("Tags");
+
+		return pendingRequestWorkBook;
+	}
+
+	private void populateSheetWithPendingRequests(Map<String, Object> allPendingRequestMap, Map<String, Object> allUserDetails, Workbook pendingRequestWorkBook) {
+		Sheet sheet = pendingRequestWorkBook.getSheetAt(0);
+		int rowCount = 0;
+		for (Map.Entry<String, Object> userEntry : allUserDetails.entrySet()) {
+			rowCount++;
+			Row row = sheet.createRow(rowCount);
+			String userId = userEntry.getKey();
+			Map<String, Object> userInfo = (Map<String, Object>) userEntry.getValue();
+			List<Map<String, Object>> pendingRequestForTheUser = (List<Map<String, Object>>) allPendingRequestMap.get(userId);
+
+			Cell nameCell = row.createCell(0);
+			Cell emailCell = row.createCell(1);
+			Cell mobileNumberCell = row.createCell(2);
+			Cell groupCell = row.createCell(3);
+			Cell designationCell = row.createCell(4);
+			nameCell.setCellValue((String)userInfo.get(Constants.FIRSTNAME));
+			emailCell.setCellValue((String)userInfo.get(Constants.PRIMARY_EMAIL));
+			mobileNumberCell.setCellValue((String)userInfo.get(Constants.MOBILE_NUMBER));
+
+			for (Map<String, Object> entry : pendingRequestForTheUser) {
+				if(entry.containsKey(Constants.GROUP)){
+					groupCell.setCellValue((String)entry.get(Constants.GROUP));
+				}
+				if(entry.containsKey(Constants.DESIGNATION)){
+					designationCell.setCellValue((String)entry.get(Constants.DESIGNATION));
+				}
+			}
+		}
+	}
+
+	private Map<String, Object> getSearchObject(Set<String> userIds, List<String> fields, Map<String, Object> filters) {
+		Map<String, Object> request = new HashMap<>();
+		Map<String, Object> requestObject = new HashMap<>();
+		request.put(Constants.FILTERS, filters);
+		request.put(Constants.FIELDS_CONSTANT, fields);
+		requestObject.put(Constants.REQUEST, request);
+		return requestObject;
+	}
+
+	private Map<String, Object> getUserSearchDetails(Set<String> userIds, boolean isMdoSearch, String rootOrgId) throws Exception {
```

Comment:
why new method -- don't we have the search user implementation already ??

Reply:
We need filtering with rootOrg Id, filters, and fields are different hence different method.